src/components/atoms/CompanyLogo/companyLogos.ts

```diff
@@ -0,0 +1,8 @@
+const companyLogos = {
+  levi: {
+    src: '',
+    alt: '',
+  },
+};
+
+export default companyLogos;
```

Comment:
I want to set a paradigm to only use default exports for files that export a singular thing.

Reply:
Agree, default exports for files that export a singular thing is a good practice that can help make the codebase more consistent